### PR TITLE
Allow specific versions for shaderc dependencies

### PR DESCRIPTION
**Pull request by softworkz**
Implements https://github.com/m-ab-s/media-autobuild_suite/issues/2367

### CODE REVIEW

File: build/media-suite_helper.sh [hunk 1]
```diff
@@ -313,6 +313,64 @@ do_vcs() {
...
+        rm -rf "$vcsFolder-git"
+        do_print_progress "  Running git clone for $vcsFolder"
+        if ! do_mabs_clone "$vcsURL" "$vcsFolder" "$ref"; then
+            echo "$vcsFolder git seems to be down"
+            echo "Try again later or <Enter> to continue"
+            do_prompt "if you're sure nothing depends on it."
+            # unset_extra_script
+            return
+        fi
+		mv "$vcsFolder-git" "$vcsFolder"
```
Comment (1480c1) [inline build/media-suite_helper.sh:354]: can you check the spacing here?

File: build/media-suite_helper.sh [hunk 2]
```diff
@@ -313,6 +313,64 @@ do_vcs() {
...
+    oldHead=$(vcs_get_merge_base "$ref")
+    do_print_progress "  Running git update for $vcsFolder"
+    log -q git.reset vcs_reset "$ref"
+    newHead=$(vcs_get_current_head "$PWD")
+
+    vcs_clean
+
+    cd ..
+	
+	return 0
```
Comment (1480c1) [inline build/media-suite_helper.sh:371]: Same here

File: build/media-suite_compile.sh [hunk 3]
```diff
@@ -1957,24 +1957,14 @@ _check=(shaderc/shaderc.h libshaderc_combined.a)
...
+        do_vcs_local "$SOURCE_REPO_GLSLANG" glslang
+        do_vcs_local "$SOURCE_REPO_SPIRV_TOOLS" spirv-tools
+        do_vcs_local "$SOURCE_REPO_SPIRV_HEADERS" spirv-headers
+        do_vcs_local "$SOURCE_REPO_SPIRV_CROSS" spirv-cross
 
-        add_third_party "$SOURCE_REPO_GLSLANG"
-        add_third_party "$SOURCE_REPO_SPIRV_TOOLS" spirv-tools
-        add_third_party "$SOURCE_REPO_SPIRV_HEADERS" spirv-headers
-        add_third_party "$SOURCE_REPO_SPIRV_CROSS" spirv-cross
+		cd ..
```
Comment (1480c1) [inline build/media-suite_compile.sh:1967]: here too

File: build/media-suite_helper.sh (hunk 1 above)
Comment (softworkz) [reply build/media-suite_helper.sh]: me==idiot

Thanks + sorry

File: build/media-suite_helper.sh (hunk 2 above)
Comment (softworkz) [reply build/media-suite_helper.sh]: Fixed

File: build/media-suite_compile.sh (hunk 3 above)
Comment (softworkz) [reply build/media-suite_compile.sh]: Fixed

**Pull request merged**